### PR TITLE
Sn/style updates

### PR DESCRIPTION
**Pull request by NakiNorton**
#### What’s this PR do?
- Adds additional styling for improved user experience
- Add conditional rendering for a 'Request for NOI submitted' message on job cards
- Change name being displayed on job cards to be the job site contact name as that's a required value so we won't have missing headings
- Fix tests that broke as a result of these changes
#### Where should the reviewer start?
- Anywhere. A lot of files were modified
#### How should this be manually tested?
- View style changes in browser
- run `npm test`
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
#75 

#### Screenshots (if appropriate)

New log out button position & larger dashboard 'buttons'
Screen Shot 2020-11-04 at 3 56 04 AM

Login is responsive:
Screen Shot 2020-11-04 at 2 28 16 AM

Job cards:
Screen Shot 2020-11-04 at 3 54 24 AM

#### Questions:
- Are the squares on the dashboard too big? I wanted them to fill up the page a bit more but I might have gone too big.
#### Need Help/Support
n/a

- [x] checked to see it passes all tests

### CODE REVIEW

File: src/helpers/apiCalls.js [hunk 1]
```diff
@@ -51,10 +51,7 @@ export const postNewJob = async (newJob, id) => {
 
 export const updateJobStatus = (id, jobId, newStatus) => {
   return fetch(`https://lienflash-be.herokuapp.com/api/v1/user/${id}/jobs/${jobId}?status=${newStatus}`, {
-    method: 'PATCH',
-    headers: {
-      'Content-Type': 'application/json'
-    }
+    method: 'PATCH'
```
Comment (tarynmartin) [inline src/helpers/apiCalls.js:54]: Did this work better?

File: src/reducers/jobInfoReducer.test.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ describe('jobInfoReducer', () => {
             business_address: '12 Tree Ave',
             business_address_line_2: 'Suite 200',
             business_city: 'Seattle',
-            business_state: 'WA', business_zip_code: '99900', additional_info: 'Amazon sucks',
+            business_state: 'WA', business_zip_code: '99900', additional_info: 'Amazon warehouse reno',
```
Comment (tarynmartin) [inline src/reducers/jobInfoReducer.test.js:34]: Thanks for cleaning up my bad content

File: src/scss/_login.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.login {
```
Comment (tarynmartin) [inline src/scss/_login.scss:1]: Thanks for styling this!

**Review by tarynmartin — APPROVED** [review]
Looks great. Looks like you may have fixed a lot of tests you fixed on your previous PR. I think the dashboard looks great - the only thing I think we should change is moving the install button/link thing down a little to give some space below the buttons

**Comment by NakiNorton** [comment]
Agreed on moving the Install down! On the desktop it was hard to see the install because I'd made the dashboard buttons so big, but I'm going to shrink them down so that'll give us more space to play with.

File: src/helpers/apiCalls.js (hunk 1 above)
Comment (NakiNorton) [reply src/helpers/apiCalls.js:54]: Yeah I did some research and it doesn't seem to make a difference to have the header in the patch requests.

**Pull request merged**